### PR TITLE
[CWM-154] Upgrade minio to the latest version.

### PR DESCRIPTION
**Pull request by iamazeem**
[CWM-154] Upgrade minio to the latest version.

Signed-off-by: Azeem Sajid <azeem.sajid@gmail.com>

### CODE REVIEW

**Comment by OriHoch** [comment]
@iamazeem please review

File: README.md [hunk 1]
```diff
@@ -45,12 +43,22 @@
 
 ## Local Development
 
+Most of the following operations require minio client binary available locally at `./mc`
```
Comment (iamazeem) [inline README.md:46]: Is it just for testing that a local instance of `mc` is required?
A system-wide `mc` installation should not be an issue, right?

**Review by iamazeem — COMMENTED** [review]
LGTM! :+1:

File: README.md (hunk 1 above)
Comment (OriHoch) [reply README.md:46]: yes, you can use a system-wide mc

**Pull request merged**